### DISCUSSION TITLE
argocd url is not visible

### DISCUSSION THREAD

**Discussion by soumitasardar** [argoproj/argo-cd#5122] [Q&A]
Hi,

We are trying to upgrade argocd with version v1.8.1  using Kustomize. but we are not able to.
We are facing with the error:
error: AccumulateTarget: rawResources failed to read Resources: Load from path ../namespace-install failed: '../namespace-install' must be a file (got d='/tmp/kustomize-122489720/manifests/ha/namespace-install')

 Can anyone please help to resolve the issue any any idea on the reason for this error.

**Comment by jannfis** [comment]
Hi, given the following `kustomization.yaml` in a given directory:

```yaml
resources:
- https://github.com/argoproj/argo-cd/manifests/ha/namespace-install?ref=v1.8.1
```

and running `kustomize build .`, the manifests are rendered correctly. Same goes for running `kustomize build .` within `manifests/ha/namespace-install` within a clone of the Argo CD repository.

How does your `kustomization.yaml` look like? What is the version of Kustomize you are using?
